.github/stale.yml

```diff
@@ -7,14 +7,14 @@ staleLabel: stale
 # Pull request specific configuration
 pulls:
   # Number of days of inactivity before an Issue or Pull Request becomes stale
-  daysUntilStale: 7
+  daysUntilStale: 14
   # Number of days of inactivity before a stale Issue or Pull Request is closed.
   # Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
-  daysUntilClose: 7
```

Comment:
I think 30 days and 14 days freshness for issues and prs are good, but if nobody cares about something becoming state, why do we need to wait for another week to close it?

Reply:
what would happen is people forgot about a PR and they need the bot ping to remind them; I suppose they could reopen too but... some notice is good imo